recipes/cssselect/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "cssselect" %}
+{% set version = "0.9.2" %}
+{% set sha256 = "76f090e40297b21fb57459b0ea73c8e46046feaf872909224d1fb861c1703ef5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: cssselect-{{ version }}.tar.gz
+  url: https://github.com/scrapy/{{ name }}/archive/v{{ version }}.tar.gz
```

Comment:
The main purpose of using jinja templates to me is to provide an abstraction (that of course comes at some cost in clarity) in order to do either or both of these things.

1. Make it easy to update.
2. Be a building block for new recipes.

Version helps 1. Checksum has some small value by making it easier to update at the top. So we have become ok with it. Name doesn't matter for 1, but helps 2. However, having it in the URL the way it is now doesn't help 1 or 2. For 1 the URL has to be removed anyway. For 2 it doesn't provide any advantage that hard-coding doesn't already satisfy. As mentioned, it already makes that URL a little bit unclear, but as it doesn't offer anything in exchange it should be dropped.